src/vaadin-combo-box.d.ts

```diff
@@ -237,22 +283,22 @@ declare class ComboBoxElement extends ElementMixin(
    */
   clearButtonVisible: boolean;
 
-  addEventListener<K extends keyof ComboBoxEventMap>(
+  addEventListener<K extends keyof ComboBoxEventMap<Item>>(
     type: K,
-    listener: (this: ComboBoxElement, ev: ComboBoxEventMap[K]) => void,
+    listener: (this: ComboBoxElement<Item>, ev: ComboBoxEventMap<Item>[K]) => void,
     options?: boolean | AddEventListenerOptions
   ): void;
 
-  removeEventListener<K extends keyof ComboBoxEventMap>(
+  removeEventListener<K extends keyof ComboBoxEventMap<Item>>(
     type: K,
-    listener: (this: ComboBoxElement, ev: ComboBoxEventMap[K]) => void,
+    listener: (this: ComboBoxElement<Item>, ev: ComboBoxEventMap<Item>[K]) => void,
     options?: boolean | EventListenerOptions
   ): void;
 }
 
 declare global {
   interface HTMLElementTagNameMap {
-    'vaadin-combo-box': ComboBoxElement;
+    'vaadin-combo-box': ComboBoxElement<any>;
```

Comment:
Could you please explain what's the main reason for using `any` instead of `unknown` here? Maybe add a comment.

It's not obvious when why some places use `any` (here, and e.g. `renderer: ComboBoxRenderer<any> | null | undefined;`) while some other places use `unknown` (e.g. `items: Array<unknown> | undefined;` and `dataProvider: ComboBoxDataProvider<unknown> | null | undefined;`).

Reply:
I had to use `any` to get rid of the  following error:

```
  Type 'ComboBoxRenderer<Item> | null | undefined' is not assignable to type 'ComboBoxRenderer<unknown> | null | undefined'.
    Type 'ComboBoxRenderer<Item>' is not assignable to type 'ComboBoxRenderer<unknown>'.
      Type 'unknown' is not assignable to type 'Item'.
        'Item' could be instantiated with an arbitrary type which could be unrelated to 'unknown'.
```

Added a comment above explaining why `any` is used for the mixin interface.